htdocs/salaries/list.php

```diff
@@ -166,6 +166,93 @@
 	$massaction = '';
 }
 
+//credit transfer request
+if ($massaction == 'withdrawrequest') {
```

Comment:
To avoid confusion,can you rename massaction = 'withdrawrequest' into 'creditransfer'